metadata_service/config.py

```diff
@@ -104,31 +99,33 @@ class LocalConfig(LocalGremlinConfig, Config):
     PROXY_ENCRYPTED = bool(distutils.util.strtobool(os.environ.get(PROXY_ENCRYPTED, 'True')))
     PROXY_VALIDATE_SSL = bool(distutils.util.strtobool(os.environ.get(PROXY_VALIDATE_SSL, 'False')))
 
-    JANUS_GRAPH_URL = None
-
     IS_STATSD_ON = bool(distutils.util.strtobool(os.environ.get(IS_STATSD_ON, 'False')))
 
     SWAGGER_ENABLED = True
-    SWAGGER_TEMPLATE_PATH = os.path.join('api', 'swagger_doc', 'template.yml')
-    SWAGGER = {
-        'openapi': '3.0.2',
-        'title': 'Metadata Service',
-        'uiversion': 3
-    }
 
 
-class NeptuneConfig(LocalGremlinConfig, Config):
+class AtlasConfig(LocalConfig):
+    PROXY_HOST = os.environ.get('PROXY_HOST', 'localhost')
```

Comment:
just a general question - since there is a several options for proxy now do we consider neo4j as default or shall we also have separate config (very similar to AtlasConfig) for neo4j ?

Reply:
I'd keep neo4j as the default one, as most of the users are still using neo4j as their backend. Also because neo4j is the default backend for Amundsen on any package repository/registry,  like PyPI, dockerhub, etc.